pom.xml

```diff
@@ -265,15 +255,18 @@
                 </executions>
             </plugin>
             <plugin>
-                <groupId>org.sonatype.plugins</groupId>
-                <artifactId>nexus-staging-maven-plugin</artifactId>
-                <version>1.7.0</version>
-                <extensions>true</extensions>
-                <configuration>
-                    <serverId>ossrh</serverId>
-                    <nexusUrl>https://oss.sonatype.org/</nexusUrl>
-                    <autoReleaseAfterClose>true</autoReleaseAfterClose>
-                </configuration>
+              <groupId>org.sonatype.central</groupId>
+              <artifactId>central-publishing-maven-plugin</artifactId>
+              <version>0.8.0</version>
+              <extensions>true</extensions>
+              <configuration>
+                <publishingServerId>central</publishingServerId>
+                <autoPublish>true</autoPublish>
+                <waitUntil>published</waitUntil>
+                <centralSnapshotsUrl>https://central.sonatype.com/api/v1/publisher/upload</centralSnapshotsUrl>
+                <username>${env.CENTRAL_TOKEN_USERNAME}</username>
```

Comment:
Do the credentials need to be set in the settings.xml file instead? Based on the central docs: https://central.sonatype.org/publish/publish-portal-maven/